src/app/components/Metadata/propTypesCheck.js

```diff
@@ -0,0 +1,17 @@
+import { checkPropTypes } from 'prop-types';
+
+const propTypeCheck = (props, propName, componentName, propType) => {
```

Comment:
I've run it and this works, but have you seen that you can call the prop type things (`string.isRequired` etc.)? You then wouldn't need to use `checkPropTypes`, which aiui is really for running in scripts and such.

https://reactjs.org/warnings/dont-call-proptypes.html#fixing-the-false-positive-in-third-party-proptypes